bin/test-lxd-network

```diff
@@ -221,6 +221,20 @@ sleep 10
 lxc list
 networkTests
 
+# Check bridged NIC type works with "agent.rename_interfaces".
+echo "=> Performing bridged NIC agent.rename_interfaces tests"
+lxc stop v1-physical
+lxc config set v1-physical agent.rename_interfaces=true
+lxc start v1-physical
+
+# Wait for lxd-agent to rename the interface.
```

Comment:
You're missing an actual sleep statement here.

Reply:
Fixed - not sure how that happened.